config/routes.rb

```diff
@@ -1,8 +1,10 @@
 Rails.application.routes.draw do
+  get 'password_resets/new'
+
   resources :articles, only: [:index, :create, :new]
   resources :users, only: [:create, :new]
   resource :session, only: [:create, :new, :destroy]
-
+  resources :password_resets
```

Comment:
You're not using the `:show` or `:destroy` methods. In those cases, I like to explicitly whitelist the routes: `resources :password_resets, only: [:new, :create, :edit, :update]`